lib/bots/syncbot.ts

```diff
@@ -358,13 +357,13 @@ export class SyncBot extends ProcBot {
      * @param type entity required
      */
     private searchPairs(event: HandleContext, type: 'room'): Promise<string> {
-        return new Promise<string>((resolve, reject) => {
+        return new Promise<string>((resolve) => {
             // Extract some indexes
             const from = event.source;
             const to = event.to;
             const id = event.sourceIds[type];
             // Try to find the value specified
-            if(
+            if (
```

Comment:
There's a newline between `if (` and first conditional (`this.rooms[from] &&`) and the final `( }`.